### ISSUE TITLE
NTR: Modifier: Necrotizing nature to vasculitis (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9321]
New term request Modifier: Necrotizing nature to vasculitis (ang_130518230952):   UMLS: 1318520

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000253
- label: Necrotizing vasculitis
- parents: [Vasculitis by histology (HP:0034785)](https://hpo.jax.org/app/browse/term/HP:0034785)
- pmid: PMID:33372616
- def: A histopathological finding characterized by inflammation of the arterial and/or capillary and/or venous blood vessels, leading to a change in the vascular wall as a whole.
- comment: Stenosis or occlusion of the vascular lumen by thrombosis or intimal proliferation is the result of endothelial damage, which is the cause of clinical manifestations. This finding may be observed with a group of diseases including polyarteritis nodosa and anti-neutrophil cytoplasmic antibody-associated vasculitis (ANCA): granulomatosis with polyangiitis, eosinophilic granulomatosis with polyangiitis, and microscopic polyangiitis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9321
- synonyms:
